src/Commands.js

```diff
@@ -37,6 +38,7 @@
             fired = [],
             prefix = "cmdId_",
             indexer = 0,
+            rethrow, //default true
```

Comment:
Why do we need it in the generic events mechanism? 
An exception in one handler should not "cut" the flow of invoking other handlers.